bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v1/ImageImpl.java

```diff
@@ -308,6 +324,16 @@ protected String getSeoFriendlyName(String imageName) {
 
     @Override
     public String getSrc() {
+        if (useWOID) {
```

Comment:
I would avoid processing this every time the method is called. I would either assign the value to `src` during initialisation of the model (as it previously was) or using a lazy/on-demand assignment and checking if the value is already assigned.